helpers/postgresql.py

```diff
@@ -39,6 +35,7 @@ def parseurl(url):
 
 
 class Postgresql:
+    _SERVER_VERSION = 90400
```

Comment:
By default we assume that we are running on 9.4
Postgres version determine default behaviour of lsn_to_bytes and bytes_to_lsn methods.

When connection to instance is established this value would be overwritten with actual value (see line 85)